en/user_guide/user.rst

```diff
@@ -0,0 +1,50 @@
+User, Group and Role
+====================
+
+Open Orchestra offer the possibility to manage different User with different right.
+
+User Management
+---------------
+
+Page Users in Administration allow to search,create,edit ans delete site users.
+
+At the creation, some fields are required :
+    - First Name
+    - Last Name
+    - E-mail
+    - Username
+    - Password
+
+After it's possible to Edit User to choose this default language which is display on connect and affiliate himself to some group.
+
+Group Management
+----------------
+
+In Open Orchestra, Users are members (or not) of groups.
+Group give at User some right and access for website administration.
+Each group have a name and is associate to ONE website.
+It contain a list of roles for User.
+
+Role Management
+---------------
+
+Role give one type of authorization for User.
+
+Role characteristics
+~~~~~~~~~~~~~~~~~~~~
+
+Roles contains a name used by sensio security bundle to restrict authorization::
+
+    /**
+     * @Security("has_role('ROLE_NAME')")
+     */
+    public function indexAction()
+    {
+        // ...
+    }
+
+
+a description for each site languages to display in admin panel and two optionals parameters :
+    - Status from
+    - Status destination
```

Comment:
* instead of -
are these actual labels from the BO ? It would rather use "Original status" "Destination status"